nominal/core/dataset.py

```diff
@@ -519,6 +519,29 @@ def _get_dataset(
     return datasets[0]
 
 
+def _create_dataset(
+    auth_header: str,
+    client: scout_catalog.CatalogService,
+    name: str,
+    *,
+    description: str | None = None,
+    labels: Sequence[str] = (),
+    properties: Mapping[str, str] | None = None,
+    workspace_rid: str | None = None,
+) -> scout_catalog.EnrichedDataset:
+    request = scout_catalog.CreateDataset(
+        name=name,
+        description=description,
+        labels=[*labels],
+        properties={} if properties is None else {**properties},
```

Comment:
```suggestion
        labels=list(labels),
        properties={} if properties is None else dict(properties),
```
functionally equivalent but this is how we do it everywhere else in the client